content/en/contribute/code/workflow.md

```diff
@@ -251,21 +253,38 @@ The ticket needs further development.
 {{< /tab >}}
 {{< /tabpane >}}
 
-A great way to facilitate discussion and collaboration is with a Draft PR.
+#### Create a branch
+
+Any code should be in a branch in each repository you update. The name of the branch should be in the form `<issue-number>-<readable-name>`, for example, `1104-inclusive-export`. 
+
+Push commits at least once a day to a remote repository, ensures that the code is always backed up and safe, protects against accidental deletes and allows team members to see the latest changes and work together more effectively.
+
+#### Pull requests
+
+Create a Draft Pull Request to facilitate discussion and collaboration with quality assistance engineers and developers.
+
+Once you are confident that the change is complete and ready to be merged:
+
+1. Change the Pull Request from `Draft` to `Ready for review`.
+2. The Pull Request title will be the commit message, it is important to follow the [commit message format]({{< ref "#commit-message-format" >}}) to name the Pull Request title properly. 
+3. Add a Pull Request description:
 
-Once you're confident that the change is complete and ready to be merged:
+   - Add a description of changes, decisions, backstory, and any extra information to facilitate review process and reduce follow-ups. 
+   - Add videos or screenshots of the tests you did before submitting the Pull Request. This increases understanding of the work, and allows the reviewers to catch any uncovered case.
+   - Add the issue number, example: `medic/cht-core#123`
 
-1. Submit a PR for each of the repositories. Each PR message and description will become the commit message and description so keep the message concise, describe what and why rather than how, and link to the issue in the description (eg: "medic/cht-core#123").
-1. Wait for the builds to succeed and ensure there are no conflicts with the the main branch so the PR can be merged.
-1. Pick one Reviewer for the PR and work with them until the code passes review. In some special cases more than one Reviewer may be necessary, but be specific about additional Reviewers and ensure you really need each of their additional reviews for a good reason. Remember, anyone can collaborate on PRs even if they aren't an official Reviewer. If you add a QA Engineer as a Reviewer, briefly comment in the ticket about what kind of testing review you expect from that engineer.
+4. Do a self-code review before asking for a review, this a good practice, almost always you will find things to fix. It saves a lot of time from you and the reviewers.
+5. Always assign a QA engineer and a developer as reviewers, they will give valuable feedback on their domain of expertise. It is okay to include one additional reviewer that has more experience in particular subject. 
```

Comment:
I want to challenge the "Always assign a QA engineer and a developer as reviewers". Always adding a QA reviewer shouldn't be necessary, and it puts a lot of pressure on our QA engineers to review _every_ PR. In an ideal setup, we should trust developers to share meaningful feedback about the testing done in a PR. I think the initial version of this bullet point is more accurate about how this process would happen: QA should be involved when the person writing the code needs QA expertise for some testing strategy. This might not always be the case, and that's fine too.

Reply:
I'm okay with that. I would like to add more clarity about this piece: 

> QA should be involved when the person writing the code needs QA expertise for some testing strategy 

When do developers add a QA as a reviewer, in your opinion? Asking for more specific scenarios
They can automate and test their work, which might make them think they don't see QA. In my experience. Quite often, they provide valuable feedback specially when it comes to their focus area (e2e / sql models, etc)

Reply:
For example:

```
Consider including a QA engineer as a reviewer when you consider you need support for major changes or new features are being introduced to the codebase, security-related changes are made, substantial user experience improvements, etc.
```

Reply:
I agree with your sentence above. QA expertise can be constructive in the following scenarios: 

- The change includes user interface changes, and a QA engineer can advise whether that needs to be considered in the current e2e suite. 
- A developer has tested things locally and needs a second person to confirm that the expected behavior occurs on another machine. In this case, the developer should specifically request: "Can you please follow these steps and documentation and confirm that it is working as expected?"
- Any major changes/new features that require significant testing. The developer is expected to have written tests, but the QA Engineer can advise on edge cases or other scenarios to consider for testing.

Reply:
Thanks! I've added some of these in that paragraph